examples/kubernetes/kernel-check/kernel-check.yaml

```diff
@@ -1,22 +1,15 @@
-apiVersion: apps/v1
-kind: Deployment
+apiVersion: batch/v1
+kind: Job
 metadata:
-  labels:
```

Comment:
nits: we can keep the labels as well :)

Reply:
yes, but I thought this might not add any real benefit/value and leaving the labels out makes this example more simple/concise. what do you think?

Reply:
yes, making sense to me :)